base/task/sequence_manager/task_queue_impl.cc

```diff
@@ -154,7 +154,7 @@ bool TaskQueueImpl::TaskRunner::PostDelayedTask(const Location& location,
                                                 OnceClosure callback,
                                                 TimeDelta delay) {
   if (!recordreplay::AreEventsDisallowed())
-    recordreplay::Assert("[RUN-1537] TaskQueueImpl::TaskRunner::PostDelayedTask %d", recordreplay::PointerId(this));
+    recordreplay::Assert("TaskQueueImpl::TaskRunner::PostDelayedTask %d", recordreplay::PointerId(this));
```

Comment:
If the assertion is no longer associated with an issue it should at least have a comment for why we're keeping it around.  Alternatively could we just delete it?

Reply:
I want to keep the `Assert` in because there is a high chance it will keep kicking up dust.
However, usually `Assert`s in this type of "hub code" high up the call stack, where a lot of unrelated features intertwine, end up falling into many different buckets.

Adding more specific filters or letting other existing filters pick up the signatures, instead of having them auto-bucketed into a (hopefully soon to be) closed bucket, is usually better for triage in our experience.

Just to give one example: [here](http://admin.replay.io/crash/controller/2122c095-cc27-44b8-aa3a-8adbef8b4087) is a crash that also hits this `Assert` but likely has to do with RTC not `WorkerThread`s.

(Semi-automatically) triaging crash signatures is hard. Any suggestions would be greatly appreciated! 😅